raphtory-benchmark/src/common/mod.rs

```diff
@@ -570,7 +571,24 @@ pub fn run_graph_ops_benches(
     );
 }
 
-fn bench_materialise<F, G>(name: &str, c: &mut Criterion, make_graph: F)
+pub fn run_proto_encode_benchmark(group: &mut BenchmarkGroup<WallTime>, graph: Graph) {
+    println!("graph: {graph}");
```

Comment:
leftover print

Reply:
I left it in on purpose because I keep forgetting how big these graphs are, can remove it if people don't like it (note this is display so it only print number of nodes, edges, and temporal edges)